packaging/android/build.sh

```diff
@@ -252,22 +252,16 @@ if [ "$QUICK" = "" ] ; then
 		# Use env to make all these temporary, so they don't pollute later builds.
 		env	PATH=$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin:$PATH \
 			CC=clang \
-			./Configure android-"$ARCH" no-ssl2 no-ssl3 no-comp no-hw no-engine no-asm --prefix="$PREFIX" -DOPENSSL_NO_UI_CONSOLE -DOPENSSL_NO_STDIO
+			./Configure shared android-"$ARCH" no-ssl2 no-ssl3 no-comp no-hw no-engine no-asm --prefix="$PREFIX" -DOPENSSL_NO_UI_CONSOLE -DOPENSSL_NO_STDIO -D__ANDROID_API__=21
```

Comment:
Why hard-code -D__ANDROID_API__=21 here? I think we should use $ANDROID_PLATFORM_LEVEL

Reply:
True.

Reply:
fixed